src/app/api/workspaces/[slug]/stakgraph/route.ts

```diff
@@ -386,6 +386,77 @@ export async function PUT(
       console.error("Failed to ensure webhook/default branch:", err);
     }
 
+    // Upsert repository and detect default branch, then ensure webhook`
+    //const poolApiKey = user?.poolApiKey;
+
+    try {
```

Comment:
put logic like this in a separate little util function (it can just be in the same file)